code/backend/pages/03_Delete_Data.py

```diff
@@ -78,7 +80,23 @@ def delete_files(files):
     st.success("Deleted files: " + str(files_to_delete))
 
 
+def get_filename(file: str) -> str:
```

Comment:
Just parsing the code in my head - will this work with @komalg1 IV work where the blob could be stored in a URI format?

Reply:
Ah good point. Let me check on that!